scripts/syntax_gen.py

```diff
@@ -493,14 +493,6 @@ def generateSyntax(builddir, alltypes, kindmap):
 
             cppf.write("}\n\n")
 
-            cppf.write("{0}* {0}::clone(BumpAllocator& alloc) const {{\n".format(k))
-            if v.final:
-                cppf.write("    return alloc.emplace<{}>(*this);\n".format(k))
-            else:
-                cppf.write("    CloneVisitor<{0}> visitor;\n".format(k))
```

Comment:
Can you also remove the CloneVisitor type from this file since it's not used now?